src/game.js

```diff
@@ -253,18 +271,30 @@
 
                 // update the timer
                 me.timer.update(time);
-
-                // clear the quadtree
-                me.collision.quadTree.clear();
-
-                // insert the world container (children) into the quadtree
-                me.collision.quadTree.insertContainer(api.world);
-
-                // update all objects (and pass the elapsed time since last frame)
-                isDirty = api.world.update(me.timer.getDelta()) || isDirty;
-
-                // update the camera/viewport
-                isDirty = api.viewport.update(me.timer.getDelta()) || isDirty;
+                
+                accumulator += me.timer.getDelta();
+                
+                updateDelta = (me.sys.interpolation) ? me.timer.getDelta() : stepSize;
+    
+                while (accumulator >= stepSize || me.sys.interpolation) {
```

Comment:
Compounding accumulators are more than the developer's problem. With this while loop, it will tank the engine.

Let's take a hypothetical update that runs for 25ms. Of course, this causes one frame (draw) to be dropped. The dropped frame causes the following update to be run twice, to "catch up" the time lost from the dropped frame. The loop runs for 25ms*2, dropping 2 additional frames. The next update will try to catch up again, running for 25ms*3, ad infinitum.

Long updates will exhibit polynomial growth in time. The cap that you just added means that the polynomial growth is capped, but at that point the game slows down, canceling any net effect from attempting to catch up.

`me.sys.interpolation` approaches this issue from the other angle; allow frames to drop, and scale velocities et al to compensate.